src/core/config/config-env.ts

```diff
@@ -55,3 +55,14 @@ export function internalTask<ArgsT extends TaskArguments>(
 }
 
 export const types = argumentTypes;
+
+export function usePlugin(name: string): void {
+  const path = process.cwd() + "/plugins/";
```

Comment:
I'm not sure about this

Reply:
I was about to comment that this restricts the plugin system too much. You can't use npm-installed plugins like this.

`require(name)` should be enough.